package.json

```diff
@@ -63,7 +63,7 @@
     "codecov": "^2.3.0",
     "eslint": "^4.9.0",
     "express": "^4.15.4",
-    "express-graphql": "^0.6.7",
+    "express-graphql": "^0.6.11",
```

Comment:
Is this dependency update required for this feature?

Reply:
No, will roll this back. Was from testing something.